src/super_gradients/examples/train_from_recipe_example/train_from_kd_recipe.py

```diff
@@ -0,0 +1,22 @@
+"""
```

Comment:
can we move this file to a new folder (module) inside examples called "train_from_kd_recipe_example"?

Reply:
Done in 5688491a77aff88cbc04113e6b79d54a8c88d8d2